### PR TITLE
Added exceptionless and foundatio projects to list

### PR DESCRIPTION
**Pull request by niemyjski**
I maintain these projects and they all support dotnet core. They are also all listed in: https://github.com/quozd/awesome-dotnet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
   * [Code Analysis and Metrics](#code-analysis-and-metrics)
   * [Compilers, Transpilers and Languages](#compilers-transpilers-and-languages)
   * [Database Drivers](#database-drivers)
+  * [Datetime](#datetime)
```
Comment (thangchung) [inline README.md]: I don't think we need to add the new section for Datetime (too specific). We have the **Internationalization** section for adding the date time already

File: README.md [hunk 2]
```diff
@@ -33,6 +36,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
   * [Logging](#logging)
   * [Mail](#mail)
   * [Mathematics](#mathematics)
+  * [Metrics](#metrics)
```
Comment (thangchung) [inline README.md]: This is the same. We have **Code Analysis and Metrics** so I don't think we need adding more **Metrics** section

File: README.md [hunk 3]
```diff
@@ -162,10 +166,19 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [RethinkDb.Driver](https://github.com/bchavez/RethinkDb.Driver) - C#/.NET RethinkDB driver with 100% ReQL API coverage.
 * [tarantool-csharp](https://github.com/progaudi/tarantool-csharp) - .NET CLI client for Tarantool NoSql database.
 
+### Datetime
+* [Exceptionless.DateTimeExtensions](https://github.com/exceptionless/Exceptionless.DateTimeExtensions) - DateTimeRange, Business Day and various DateTime, DateTimeOffset, TimeSpan extension methods.
+
+### Distributed Computing
```
Comment (thangchung) [inline README.md]: Actually, we don't have many items for **distributed computing** on this list so I think we should put it into the **Misc**

**Comment by thangchung** [comment]
@niemyjski I really thank for your PR into this list. Really awesome libs for logging. I have some comments for you 👍

File: README.md (hunk 1 above)
Comment (niemyjski) [inline README.md]: This isn't i8n tho,

File: README.md (hunk 2 above)
Comment (niemyjski) [inline README.md]: There are other metrics apps like app-metrics.io which works on core.

File: README.md (hunk 3 above)
Comment (niemyjski) [inline README.md]: There are others, I just only added ours due to pr requirements

**Comment by niemyjski** [comment]
Thanks! I added some comments :)

File: README.md (hunk 2 above)
Comment (thangchung) [inline README.md]: I don't know why these is different :)

File: README.md (hunk 1 above)
Comment (thangchung) [inline README.md]: Okay. I agree with your point.

File: README.md (hunk 2 above)
Comment (niemyjski) [inline README.md]: Should I put this under that?

Comment (thangchung) [inline README.md]: Yes. I think so. We will have a look and change it later

**Comment by niemyjski** [comment]
@thangchung I've made the requested changes.

File: README.md [hunk 4]
```diff
@@ -202,6 +215,8 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 ### Logging
 * [common-logging](https://github.com/net-commons/common-logging) - Portable logging abstraction for .NET [http://net-commons.github.io/common-logging](http://net-commons.github.io/common-logging).
 * [dnxcore-logging-logstash](https://github.com/jvandevelde/dnxcore-logging-logstash) - Logstash logging extension for .NET Core applications with UDP and Redis transports.
+* [Foundatio](https://github.com/exceptionless/Foundatio#logging) - A fluent logging api that can be used to log messages throughout your application.
+* [Exceptionless](https://github.com/exceptionless/Exceptionless.Net) - Exceptionless .NET Client
```
Comment (thangchung) [inline README.md]: I think this one should move upper to Foundatio lib. Thanks

Comment (niemyjski) [inline README.md]: What do you mean (in alpha order)?

Comment (niemyjski) [inline README.md]: Updated

**Comment by niemyjski** [comment]
Any ideas on when this would be merged in? Now there are some conflicts...

**Comment by thangchung** [comment]
I have reviewed and resolved your changes into the changes from this repo. Thank you for your patience to wait for me for a long time 👍

**Pull request merged**